tests/components/myq/test_binary_sensor.py

```diff
@@ -0,0 +1,20 @@
+"""The scene tests for the myq platform."""
+
+from homeassistant.const import STATE_ON
+
+from .util import async_init_integration
+
+
+async def test_create_binary_sensors(hass):
+    """Test creation of binary_sensors."""
+
+    await async_init_integration(hass)
+
+    state = hass.states.get("binary_sensor.happy_place_myq_gateway")
+    assert state.state == STATE_ON
+    expected_attributes = {"device_class": "connectivity"}
+    # Only test for a subset of attributes in case
+    # HA changes the implementation and a new one appears
+    assert all(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5281369)